manifests/A Coalition at War.yaml

```
@@ -0,0 +1,17 @@
+name: A Coalition at War
+authors: Josh Mudge
+homepage: https://github.com/mathwhiz1212/A-Coalition-At-War
+license: GPL-3.0-or-later
+version: v0.0.1a # Changed automatically
+shortDescription: See the conflict between the Coalition and the Quarg. The story could use some love if you have some to give.
+description: >-
+You hear a lot about the conflict between the Coalition and the Quarg in Endless Sky, but you never see it.
+This plugin introduces a storyline that lets you see that conflict and even capture Heliarch ships later on.
+You can start the only mission by landing on "Tebuteb's Table" in the Tebuteb system and accept the anamoly mission.
+Please consider helping to add more misisons. What should be included? What stories should be told? https://github.com/mathwhiz1212/A-Coalition-At-War/issues/new/choose
+url: https://github.com/mathwhiz1212/A-Coalition-At-War/archive/refs/tags/v0.0.1a.zip
+iconUrl: https://raw.githubusercontent.com/endless-sky/endless-sky/master/images/outfit/unknown.png
+autoupdate:
+  type: tag
+  url: https://github.com/mathwhiz1212/A-Coalition-At-War/archive/refs/tags/$version.zip
+  iconUrl: https://raw.githubusercontent.com/endless-sky/endless-sky/master/images/outfit/unknown.png # N/A
```

iconUrl should be versioned as well
```suggestion
  iconUrl: https://raw.githubusercontent.com/endless-sky/endless-sky/$version/images/outfit/unknown.png # N/A
```

---

I don't have a logo. I just linked to a static "unknown outfit" icon from the main Endless Sky repo because I noticed that's what you used for other plugins that don't have a logo.

I'll just remove references to an icon because that's what other no logo plugins do.

---

https://github.com/mathwhiz1212/endless-sky-plugins/commit/8c682d1573273e263806e3ff5eb05af21e0fdec5 should resolve this.